bears/vcs/git/GitCommitBear.py

```diff
@@ -110,6 +114,38 @@ def check_shortlog(self,
                     self,
                     "Shortlog of HEAD commit does not match given regex.")
 
+        if shortlog_imperative_check:
+            colon_pos = shortlog.find(':')
+            shortlog = shortlog[colon_pos:] if colon_pos != -1 else shortlog
```

Comment:
Thankyou didn't notice that. Awesome. :+1: :D
ᐧ

On Mon, Apr 4, 2016 at 11:23 PM, Mischa Krüger <notifications@github.com>
wrote:

> In bears/vcs/git/GitCommitBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/315#discussion_r58417802>
> :
>
> > @@ -110,6 +114,38 @@ def check_shortlog(self,
> >                      self,
> >                      "Shortlog of HEAD commit does not match given regex.")
> >
> > +        if shortlog_imperative_check:
> > +            colon_pos = shortlog.find(':')
> > +            shortlog = shortlog[colon_pos:] if colon_pos != -1 else shortlog
>
> shortlog[colon_pos:] extracts the colon too, shouldn't it be shortlog[colon_pos
> + 1:]?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/315/files/23d9af66bcb498b19adffe8effb5af1ec8aa4f40#r58417802>
>